ibis-server/app/model/metadata/postgres.py

```diff
@@ -71,15 +81,38 @@ def get_table_list(self) -> list[Table]:
                 )
 
             # table exists, and add column to the table
-            unique_tables[schema_table].columns.append(
-                Column(
-                    name=row["column_name"],
-                    type=self._transform_postgres_column_type(row["data_type"]),
-                    notNull=row["is_nullable"].lower() == "no",
-                    description=row["column_comment"],
+            column = Column(
+                name=row["column_name"],
+                type=self._transform_postgres_column_type(row["data_type"]),
+                notNull=row["is_nullable"].lower() == "no",
+                description=row["column_comment"],
+                properties=None,
+            )
+            unique_tables[schema_table].columns.append(column)
+
+            if row["constraint_type"] == "PRIMARY KEY":
+                if schema_table in unique_tables_primary_key:
+                    unique_tables_primary_key[schema_table].append(column)
+                else:
+                    unique_tables_primary_key[schema_table] = [column]
+
+        for schema_table, columns in unique_tables_primary_key.items():
+            if len(columns) == 0:
+                continue
+            elif len(columns) == 1:
+                unique_tables[schema_table].primaryKey = columns[0].name
+            else:
+                composite_primary_key_column = Column(
+                    name="composed_primary_key",
+                    type="json",
+                    notNull=True,
+                    description=f"Composed primary key based on fields: {[col.name for col in columns]}",
                     properties=None,
+                    nestedColumns=columns,
                 )
```

Comment:
Is it possible to have multiple primary keys in the same table? IMO, the postgres only allows the table has only one primary key. Given a table `customer`:
```
test=# alter table customer add constraint cky primary key (c_custkey);
ALTER TABLE
test=# alter table customer add constraint cky primary key (c_name);
ERROR:  multiple primary keys for table "customer" are not allowed
```